source/edge-server/mongodb.txt

```diff
@@ -10,4 +10,100 @@ Edge Server MongoDB API Support
    :depth: 2
    :class: singlecol
 
-This is a placeholder page.
+Atlas Edge Server natively implements a subset of the MongoDB wire protocol,
+which allows you to connect to an App through one of its linked MongoDB 
```

Comment:
which allows you to access your Atlas data on the edge using standard MongoDB drivers and tools